internal/services/formatters/ruby/deployments/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ruby:3.0.2-alpine
+FROM ruby:2.4.10-alpine
```

Comment:
This will still working for users that use >=3.0 ?

Reply:
was added in commentary
```
Was Update dockerfile of Ruby language for usage ruby:2.4.10-alpine base image, because brakeman tool is not suport for newest version of ruby
```

Reply:
Oh, sorry!!